README.md

```diff
@@ -27,7 +27,19 @@ This plugin uses familiar Kubernetes and container tools to bring you a rich Kub
 
 ## Installing the plugin into your IDE
 
-(coming soon)
+If you'd like to try out the plugin pre-release, you can build it from source and install it into your IDE:
+
+1) Clone this repository to your machine:<br />
+   `git clone git@github.com:GoogleContainerTools/google-container-tools-intellij.git`
+2) Build the plugin from the root of the repository:<br />
+   `./gradlew buildPlugin`
+3) Find the `.zip` binary in the root of the repository under: <br />
+   `build/distributions/google-container-tools-intellij.zip`
+4) Install the plugin into your JetBrains IDE:
+   1) Go to `Preferences/Settings > Plugins`
+   2) Select the settings gear icon and then `Install Plugin From Disk`:
+      ![manual-install](src/main/resources/images/plugin-manual-install.png =500x) 
```

Comment:
Should we put the documentation images into sources? What if we put them separately, maybe in `docs/images` or something like that? It feels we usually put something used by code into `resources`.